source/assets/public/index.css

```diff
@@ -56,10 +55,8 @@
  */
 
 html {
-  font-size: 62.5%;
-}
-.breakpoint-phone {
   font-size: 52.5%;
+  overflow-x: hidden;
```

Comment:
Why `overflow-x: hidden`?

Reply:
Some elements of the page contents were extending out of the container at small breakpoints.